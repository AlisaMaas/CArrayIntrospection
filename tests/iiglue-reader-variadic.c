#include "iiglue-reader-variadic.h"

void variadic(int *vector, ...) {
	vector[3] = 7;
}
