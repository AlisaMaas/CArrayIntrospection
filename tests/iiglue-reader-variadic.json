{"libraryEnums":[],"libraryFunctions":[{"foreignFunctionLinkage":"LinkDefault","foreignFunctionParameters":[{"parameterType":{"contents":{"contents":32,"tag":"CInt"},"tag":"CPointer"},"parameterName":"vector","parameterAnnotations":[{"contents":[],"tag":"PANotNull"},{"contents":1,"tag":"PAArray"}]}],"foreignFunctionReturnType":{"contents":[],"tag":"CVoid"},"foreignFunctionName":"variadic","foreignFunctionAnnotations":[]}],"libraryName":"iiglue-reader-variadic","libraryTypes":[],"libraryDependencies":[],"libraryAnnotations":[]}