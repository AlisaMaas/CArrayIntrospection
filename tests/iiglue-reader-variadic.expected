Printing analysis 'Read iiglue analysis results and add them as metadata on corresponding LLVM entities':
	array arguments:
		variadic::vector
